commands/cat.py

```diff
@@ -0,0 +1,26 @@
+from db.dao.directory_dao import DirectoryDao
+from base.command import BaseCommand
+from base.arguments import StringArgument
+from db.dao.file_dao import FileDao
+
+
+class CatCommand(BaseCommand):
+    command = 'cat'
+    arguments = [StringArgument()]
+
+    def run(self):
+        path = self.arguments[0].data
+        file = self.context.parse(path, True)
+        file_directory = DirectoryDao.get_directory_from_directory_id(
+            file.directory_id)
+
+        if (FileDao.is_unique_filename(file.name, file_directory)
```

Comment:
We do not need these checks, `context.parse` will raise errors if the file does not exist.